tools/releaseBuild/build.json

```diff
@@ -14,6 +14,19 @@
       "DockerImageName": "ps-snap-ubunutu-16-04",
       "BinaryBucket": "release"
     },
+    {
+      "Name": "powershell-snap-stable",
```

Comment:
adding a build name similar to the preview name to make it easier to calculate the build names.